src/scripts.js

```diff
@@ -200,32 +208,58 @@ function handleSingleRecipeButtons(event) {
   } else if (event.target.innerText === 'Add To Favorites') {
     addRecipeToFavorites(currentRecipe);
   } else if (event.target.innerText === 'Add To Cook') {
-    currentUser.addToCookList(currentRecipe);
+    displayRecipeToCook();
   }
 }
 
-//IDEA for how to dynamically add radio buttons that represent 
-//the recipe the user chooses to cook
-//
-//cook button = document.getElementById('click'), function() {
-//   var cookItem = document.createElement('input');
-//   cookItem.type = 'radio';
-//   cookItem.id = 'cook${recipeid}'; //need to have an id that doesn't start with a number...so put cook behind each unique id??
-//   cookItem.value = '${recipe.name';
- 
-//   var label = document.createElement('label')
-//   label.htmlFor = 'cook${recipe.id';
- 
-//   var description = document.createTextNode('Email');
-//   label.appendChild(description);
- 
-//   var newline = document.createElement('br');
- 
-//   var container = document.getElementById('container');
-//   container.appendChild(radiobox);
-//   container.appendChild(label);
-//   container.appendChild(newline);
-// }
+function displayRecipeToCook() {
+  currentUser.addToCookList(currentRecipe);
+  pantryCookList.innerHTML += ` <li class="pantry-cook-item">
+  <input class ="pantry-btn" type="radio" id="cook${currentRecipe.id}" name="cook-recipe" value="${currentRecipe.name}" />
+<label class="pantry-cook-item-label" for="${currentRecipe.name}">${currentRecipe.name}</label>
+</li>`
+}
+
+function handleCookThisButton() {
+  let recipeSelection;
+  const cookList = document.querySelectorAll(".pantry-btn")
+  cookList.forEach(recipe => {
+    if (recipe.checked) {
+      recipeSelection = recipesRepo.filterRecipesByName(recipe.value);
+    } else {
+      //What should happen here
+      console.log("Please make a selection");
+    }
```

Comment:
I was thinking here, if none of the inputs are checked, we could put some code here to manipulate the DOM to display a message to the user...like "Please select a recipe to cook"

Reply:
Yeah I was thinking the same thing. I like it!